ironic-provisioning-log-watch.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/bash
+
+# Ramdisk logs path
+LOG_DIR="/shared/log/ironic/deploy"
+
+until  ls "${LOG_DIR}"/*.tar.gz 1> /dev/null 2>&1
+do
+    sleep 1
+done
+
+cd "${LOG_DIR}"
+
+for fn in *.tar.gz
+do
+    echo "************ Contents of $fn ramdisk log file bundle **************"
+    tar -xOzvvf $fn | sed -e "s/^/$fn: /"
+done
+
+rm -rf ${LOG_DIR}/*
```

Comment:
This is potentially racy, maybe track the files we've already dumped and remove only them?

Reply:
you're also still in the LOG_DIR, the filesystem may not like that
maybe execute the for loop on L13 from outside ?

Reply:
Moved rm inside the loop.